app/views/ticket_mailer/new_ticket.erb

```diff
@@ -0,0 +1,2 @@
+Your ticket number: <%= @ticket_id %> was created. <BR><HR>
```

Comment:
will this ticket system support _plain-text_ email responses besides HTML or is out of scope to set multipart emails? Seems considered a good practice to have it.

Reply:
Yeah. It's good practice. In this case, I think it's not necessary because the goal is to understand Inbound emails, not outbound 😄